src/platform/Ameba/SystemTimeSupport.cpp

```diff
@@ -58,26 +58,32 @@ Milliseconds64 ClockImpl::GetMonotonicMilliseconds64(void)
 
 CHIP_ERROR ClockImpl::GetClock_RealTime(Clock::Microseconds64 & curTime)
 {
-    // TODO(19081): This platform does not properly error out if wall clock has
-    //              not been set.  For now, short circuit this.
-    return CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE;
-#if 0
-    time_t seconds;
-    struct rtkTimeVal tv;
+#if defined(CONFIG_ENABLE_AMEBA_SNTP) && (CONFIG_ENABLE_AMEBA_SNTP == 1)
```

Comment:
same here: this should be `#if CONFIG_ENABLE_AMEBA_SNTP` with a proper include that guarantees this define is available or not.

Reply:
Yes, it is actually already defined in config/ameba/chip.cmake. Updated in the last commit.